fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -825,27 +825,29 @@ public void testAgentAsUri() throws IOException {
         final String id = "/rest/" + getRandomUniqueId();
         final String testObj = ingestObj(id);
 
-        logger.debug("Anonymous can't read (no ACL): {}", id);
+        logger.debug("Anonymous can read (has ACL:READ): {}", id);
         final HttpGet requestGet1 = getObjMethod(id);
-        assertEquals(HttpStatus.SC_FORBIDDEN, getStatus(requestGet1));
+        assertEquals(HttpStatus.SC_OK, getStatus(requestGet1));
 
         logger.debug("Can username 'smith123' read {} (no ACL)", id);
         final HttpGet requestGet2 = getObjMethod(id);
         setAuth(requestGet2, "smith123");
-        assertEquals(HttpStatus.SC_FORBIDDEN, getStatus(requestGet2));
+        assertEquals(HttpStatus.SC_OK, getStatus(requestGet2));
 
         System.setProperty(USER_AGENT_BASE_URI_PROPERTY, "info:user/");
         System.setProperty(GROUP_AGENT_BASE_URI_PROPERTY, "info:group/");
 
         logger.debug("Can username 'smith123' read {} (overridden system ACL)", id);
         final HttpGet requestGet3 = getObjMethod(id);
         setAuth(requestGet3, "smith123");
-        assertEquals(HttpStatus.SC_FORBIDDEN, getStatus(requestGet3));
+        // robyj - fcrepo-2668 changed this from SC_FORBIDDEN to SC_OK
+        assertEquals(HttpStatus.SC_OK, getStatus(requestGet3));
 
         logger.debug("Can username 'group123' read {} (overridden system ACL)", id);
         final HttpGet requestGet4 = getObjMethod(id);
         setAuth(requestGet4, "group123");
-        assertEquals(HttpStatus.SC_FORBIDDEN, getStatus(requestGet4));
+        // robyj - fcrepo-2668 changed the code from SC_FORBIDDEN to SC_OK
```

Comment:
Comment can be removed.

Reply:
I believe i've made those modifications in the commit I just made